_layouts/base.html

```diff
@@ -153,7 +153,13 @@ <h3 class="usa-footer__logo-heading">U.S. General Services Administration</h3>
     <script src="{{ site.baseurl }}/assets/js/main.js"></script>
     <script src="{{ site.baseurl }}/assets/js/anchor.js"></script>
     <script>
-      anchors.add('#main-content h1:not(.usa-accordion__heading), #main-content h2, #main-content h3, #main-content h4, #main-content [data-anchor]');
+      anchors.add([
+        '#main-content h1:not(.usa-accordion__heading)',
+        '#main-content h2:not(.usa-accordion__heading)',
+        '#main-content h3:not(.usa-accordion__heading)',
+        '#main-content h4:not(.usa-accordion__heading)',
+        '#main-content [data-anchor]'
```

Comment:
Existed previously, but I couldn't find anywhere this is used? Could we remove?

Reply:
I guess so, we can always bring it back when we need it. removed in 909b25d65f13c7638eae50675047dec5faa81778